tokenizer/gpt2_tokenizer.hpp

```diff
@@ -0,0 +1,26 @@
+#include <kernels/ustring.hpp>
```

Comment:
What's the idea of extracting this header file?

Reply:
- Keep the same format with other ops
- Let GPT2Tokenizer could be registered by `operator_lists`
I noticed that GPT2Tokenizer is registered by `LoadTokenizerSchemaList`, should we keep this registration way?

Reply:
> * Keep the same format with other ops
> * Let GPT2Tokenizer could be registered by `operator_lists`
>   I noticed that GPT2Tokenizer is registered by `LoadTokenizerSchemaList`, should we keep this registration way?

The operator list will bring a lot of trouble if we want to use ifdef to enable/disable the operator because of the header file dependency. We should create some functions like LoadTokenizerSchemaList to decouple all operators.

Reply:
I think header won't bring trouble because compiler will remove those function/class are not referred by the exported functions if we set right option.

```c++
#include "GPT2Tokenizer.hpp"

class GPT2Tokenizer m_gpt2;

auto register_list = [
#ifdef INCLUDE_GPT2
 &m_gpt2
#endif
]
```
If we turn off `INCLUDE_GPT2`, the whole GPT2Tokenizer will not be included into the final binary, even it's included in a cpp. Compiler will think class GPT2Tokenizer is useless and remove it freely, because there is not any import code(exported function) refer it.

Reply:
GCC-liked library is a little complicated than you think. And In the most case, there are a lot of dependency file included in the header file, if there is a third party library, it will introduces a lot of ifdef in the file.

So the load function is the simple way to do it and prevent the user from missing ifdef in the file. And this library, there is no need to let the operator referenced by each other, and no need for the header file at all.

Reply:
I move the content of header into cpp.